src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User[] users = UserService.getUsers();
+        for (User user : users) {
+            if (email.equals(user.getEmail())
+                    && password.equals(user.getPassword())) {
+                return true;
+            }
+        }
         return false;
```

Comment:
we've implemented already that logic in the UserService, so why create duplicating code?
use UserService for finding user by email, and do required check with that user